editor/src/components/canvas/controls/namerowcrumbs.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from 'react'
```

Comment:
It looks like this was moved as well as modifications made to it, would it be possible for the original path to be used  please? Just because I've also been making changes to this file and it'll potentially make any merge I have to do quite painful.